app/routes/languages.rb

```diff
@@ -10,6 +10,13 @@ class Languages < Core
         erb :"languages/index", locals: { active: active, inactive: inactive }
       end
 
+      get '/repository' do
+        tracks = X::Track.all
+        active, inactive = tracks.partition(&:active?)
```

Comment:
It looks like we also want to partition the inactive into upcoming and planned. Let's do that here rather than in the view.